security/secret-management.md

```diff
@@ -0,0 +1,108 @@
+3. **Determine the Type**: Identify whether the leaked secret is a production secret, a test secret or a non-secret key.
+4. **Assess the Impact**: Evaluate the severity and potential impact of the leak on systems, data, and user security.
+5. **Open Source**: Identify whether the project is Open Source or not. Open Source projects
+6. **Report**: Report the issue to your technical leader to make sure that the issue has been dealt correctly. They 
+   might help to validate that the proper action has been or will be performed
+   
+### Remediation
+If the secret relates to an Open Source Repository
+1. **Revoke the Secret**: Immediately revoke the leaked secret to prevent unauthorized access. For instance, rotate API 
+   keys, reset passwords, or invalidate tokens.
+2. **Update Configuration**: Replace the leaked secret with a new, secure value in all relevant systems and codebases.
+3. **Remove from History**: Ensure the secret is removed from the code repository history. Use tools like 
+ _git filter-branch_ or _BFG Repo-Cleaner_ to clean the repository history.
+
+
+### Report
+1. **Document the Incident**: Record details of the leak in GitGuardian, including the type of secret, the steps taken 
+   to remediate it, and any lessons learned. 
+2. **Implement Preventive Measures**: Update security practices, tools, and workflows to prevent similar incidents in 
+   the future. This may include enhancing code review practices, improving secret management practices, or increasing 
+   training for developers.
+
+## Best Practices for Secret Management
+
+   **Environment Variables**: Store secrets in environment variables instead of hardcoding them in the source code or
+   properties files
+   **Secret Management Tools**: Utilize dedicated secret management tools like AWS Secrets Manager to manage and store 
+   secrets securely.
+   **Principle of least privilege**: Ensure that only necessary services and users have access to secrets.
+
+
+## Responsibility for Secret Management
+
+To ensure the security and integrity of our secrets, the Web Operations team will be responsible for the storage and 
+management of all production and development secrets. This centralized approach helps maintain control over access and 
+ensures that only authorized personnel have the ability to manage and access sensitive information.
+
+### Responsibilities of the Operations Team:
+
+- **Secure Storage**: Manage and store all production and test secrets using secure and approved secret management tools 
+  and practices.
+- **Issue Secrets**: Create new secrets when access to sensible systems is needed. This responsibility can be delegated
+  to helpdesk for some systems.
+- **Access Control**: Implement strict access controls and policies to ensure that only authorized team members and 
+  systems can access production secrets. Regularly review and update access permissions. Test secrets should be 
```

Comment:
Can we set a timeframe for this? Eg, "Every three months"

Reply:
This is probably something we need to negociate with both WebOps and helpdesk as this responsibility is delegated for some systems (i.e. CMS, Attlasian Suite, ...). Production secrets actually guarded by WebOps are very rarely shared.

Reply:
Lets go with 3 months to start with, it's easily changed and amended as we see fit

Reply:
@jameshaddrillvs, @allanhenderson I will set the timefrime in the Audit section and not here.

I just noticed that the following sentence "Regularly review and update access permissions." means essentially to conduct the audit soI will remove that from the point